plugins/ps/app-restart

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
why did this change? seems unrelated to config....

Reply:
The old config plugin used a direct execution of `ps_restart` if an app's config had been changed and `--no-restart` wasn't set. The easiest way I could find to get to this functionality from the new go-based config plugin is through a plugn trigger.